cli/cmd/plugin/ui/web/tanzu-ui/src/views/workload-cluster/select-management-cluster.scss

```diff
@@ -7,3 +7,26 @@
 .header-mc-grid {
   background-color: #222222;
 }
+
+.cc-description-text {
+  font-size: var(--cds-global-typography-font-size-3);
+  color: var(--cds-global-color-white);
+}
+
+.cc-parent-card {
```

Comment:
I have a css class in index.scss called 'section-raised' that might be worth trying and see if you like how it looks in the context of this UI. I should probably document the usage of that class a little bit, and also open to renaming the class. If you don't like how it looks in this area of the UI, please disregard. 

I would also maybe consider moving the accordion styles to index.scss so that these styles can be consumed in other areas of the app. They could be renamed to drop the 'cc-' prefix if so.

Reply:
Works for me